lib/benchmark_driver/ruby_interface.rb

```diff
@@ -66,11 +61,19 @@ def compare!
     end
 
     def rbenv(*versions)
-      versions.each do |version|
-        @executables << BenchmarkDriver::Rbenv.parse_spec(version)
+      @config.executables.replace versions.map do |version|
```

Comment:
Why would we need to reset this? I'm expecting the use case like

```
x.rbenv '2.3.0'
x.rbenv '2.4.0'
```

so this is a breaking change.

Reply:
Please just don't touch the `#run` and `#rbenv`, I don't think the change is needed for supporting `add_executable`

Reply:
I think `add` means adding executables into the default target executable.
This is why I rewrite (remove) `@executables` and adding `reset_executables`.
(I missed that `rbenv` method can be appeared multiple times)

However, it is also reasonable that we recommend to specify all of executables explicitly.
If so, we don't need to use `add_` prefix, just `executable name: ..., command: ...`.

I'm not sure we need to allow Array or String on `command:` argument.